### ISSUE TITLE
Sound on different events

### ISSUE THREAD

**Issue by fmeringdal** [fmeringdal/nettu-meet#6]
It would greatly improve the UX if the frontend played some kind of sound on events that happened in the meeting:

- Participant joined (if number of participants is less than 5)
- Chat message received
- File uploaded
- Etc

**Comment by fmeringdal** [comment]
Hi @mayankbucha !
That would be great.
First of all, are you be able to install WSL (windows subsystem for linux)? That would make the development on this project much smoother.

**Comment by mayankbucha** [comment]
Yes, I have installed WSL 2 and docker.

**Comment by fmeringdal** [comment]
Great! Have you tried to run the commands specified in the README for how to setup this project? Is that working for you, or did you get any errors?

**Comment by mayankbucha** [comment]
While executing npm i, I am getting errors for clang-tools-prebuilt.
![Screenshot 2021-06-11 001159](https://user-images.githubusercontent.com/47553721/121579869-c1eead00-ca49-11eb-85bb-759c15ede512.png)

**Comment by mayankbucha** [comment]
After executing the command in WSL shell, I am facing these issues.
![Screenshot 2021-06-11 010223](https://user-images.githubusercontent.com/47553721/121586056-d5514680-ca50-11eb-98f7-05721d58cae8.png)

**Comment by fmeringdal** [comment]
Yep, you ned to install `npm` on the new system. Try following this [article](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl).

**Comment by mayankbucha** [comment]
Hi, I have installed node and npm on wsl. When running npm start, I am getting bad variable name error:
![image](https://user-images.githubusercontent.com/47553721/121726495-430c7980-cb08-11eb-8549-4b24635fb341.png)

**Comment by fmeringdal** [comment]
Hi,

It is very difficult for me to know why this fails on `WSL` as I don't use it and can't test it. If the problem persists I suggest adding all the variables in the `.env` file to your environment variables either manually or through a powershell script and then run `./node_modules/.bin/ts-node-dev src/index.ts`

**Comment by mayankbucha** [comment]
Hi @fmeringdal, I have managed to run the mediasoup and Redis server. But, while running MongoDB, I am facing issues. So, do I need to configure MongoDB for nettu-meet in wsl.
![Screenshot 2021-06-13 182431](https://user-images.githubusercontent.com/47553721/122108710-263eb180-ce3a-11eb-95a5-1c63b74c5442.png)

**Comment by fmeringdal** [comment]
Try to change `MONGODB_NAME` from `test` to `nettu-meeting` in your `.env` file
